.vsts-ci.yml

```diff
@@ -112,11 +111,16 @@ extends:
               inputs:
                 sourceFolder: 'artifacts/packages/$(_BuildConfig)/Shipping/'
                 contents: '*.msi'
-                targetFolder: '$(Build.ArtifactStagingDirectory)'
+                targetFolder: '$(Build.ArtifactStagingDirectory)\artifacts'
```

Comment:
Added artifacts dir to isolate from other files when doing signing verification in the next step